docs/docs_source/build_test_ship/environments.rst

```diff
@@ -0,0 +1,65 @@
+
+..  _environments:
+
+Office Online environments
+==========================
+
+..  spelling::
+
+    Dogfood
+
+Office Online provides two environments for use by cloud storage partners. The :ref:`discovery URLs` for each
+environment are provided below.
+
+Initially you'll be given access to the the test environment (also called 'Dogfood'), and you should use that
+environment when building and testing your integration.
+
+Once you believe you're ready to release your integration, you can start the :ref:`'Go Live' process <go live>`. As
+part of that process, your application will be given access to the production environment.
+
+
+..  _dogfood:
+..  _test environment:
+
+Test environment
+----------------
+
+The test environment is updated frequently - usually at least once a day - and all initial testing should be done
+against it. Initially this is the only environment you will be able to access.
+
+The test environment runs the most recent Office Online code available. This does mean that you may see features
+there that are not yet available in production. However, the WOPI interactions between Office Online and your WOPI
```

Comment:
Should we note that they should not take screenshots from dogfood or publicly discuss features there?  Do the contracts they sign with us include an NDA and should we reference it here?